frontend/test/metabase/scenarios/question/custom_column.cy.spec.js

```diff
@@ -492,24 +492,26 @@ describe("scenarios > question > custom columns", () => {
     cy.button("Done").should("not.be.disabled");
   });
 
-  describe.skip("contentedtable field (metabase#15734)", () => {
+  describe("contentedtable field (metabase#15734)", () => {
     beforeEach(() => {
       // This is the default screen size but we need it explicitly set for this test because of the resize later on
       cy.viewport(1280, 800);
 
       openOrdersTable({ mode: "notebook" });
       cy.findByText("Custom column").click();
-      popover().within(() => {
-        cy.get("[contenteditable='true']")
+
+      cy.get("[contenteditable='true']")
           .as("formula")
-          .type("1+1")
-          .blur();
+
+      popover().within(() => {
+        _typeUsingGet("[contenteditable='true']", "1 + 1", 400);
```

Comment:
I wouldn't change the original repros here.
The whole point of the "fix" should be that this works without any additional hacks like `_typeUsingGet()`. WE should get rid of this function when we finally fix this.

Also, if you need to "ease" race conditions, you can try to use a "slight" delay in the type function like this:
`whateverYieldedFromThePreviousCommand().type("Foo", { delay: 50 });`

Reply:
This SHOULD work if you merge (cherry-pick) the fixes from the `master`.

Reply:
Having discussed this with @nemanjaglumac, for now we are going with `_typeUsingGet` and `_typeUsingPlaceholder` here.

Reason is even in master we need them in the tests.

We now have a separate Github issue to try and eliminate the need for, or at least simplify, these workaround functions: https://github.com/metabase/metabase/issues/16131